caffe2/onnx/backend.cc

```diff
@@ -941,9 +939,14 @@ void Caffe2Backend::OnnxToCaffe2(
     const std::vector<Caffe2Ops>& extras) {
   auto device_option = GetDeviceOption(Device(device));
 
+#if !CAFFE2_MOBILE
```

Comment:
well, rather than having crippled functionality, should we just say that OnnxToCaffe2 does not exist at all on mobile?